lib/membrane_flac_plugin/membrane_flac_parser.ex

```diff
@@ -38,18 +47,43 @@ defmodule Membrane.FLAC.Parser do
     {[], state}
   end
 
+  defp calculate_pts(buffer, state) do
```

Comment:
It should be rather called something like `set_pts`. Also, let's move it under `handle_buffer`